src/Stratis.Bitcoin.Features.Wallet/Controllers/LegacyExtPubKeyConverter.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Linq;
+using NBitcoin;
+using NBitcoin.DataEncoders;
+
+namespace Stratis.Bitcoin.Features.Wallet.Controllers
+{
+    /// <summary>
+    /// Stratis X has a version prefix starting xq5h. Full node has a version prefix xpub for extended public key. 
+    /// 3rd parties like Ledger haven't update yet to the full node version prefix.
+    /// This class provides a way to convert from the legacy format to the current format.
+    /// </summary>
+    /// <remarks>This class can be removed when the 3rd parties update their software.</remarks>
+    public class LegacyExtPubKeyConverter
```

Comment:
What's the reason for having this class to be static?

Reply:
The method is static because it was just manipulating data, it could just as easily be a string extension